pkg/customers/approval_test.go

```diff
@@ -137,42 +139,126 @@ func TestCustomers__containsValidPrimaryAddress(t *testing.T) {
 }
 
 func TestCustomers__updateCustomerAddress(t *testing.T) {
```

Comment:
These tests are definitely getting pretty long; future refactors and helper methods should cut down a lot of boilerplate.